src/expr.rs

```diff
@@ -1336,9 +1336,7 @@ fn rewrite_call_inner<R>(context: &RewriteContext,
     // Replace the stub with the full overflowing last argument if the rewrite
     // succeeded and its first line fits with the other arguments.
     match (overflow_last, tactic, placeholder) {
-        (true,
-         DefinitiveListTactic::Horizontal,
-         placeholder @ Some(..)) => {
+        (true, DefinitiveListTactic::Horizontal, placeholder @ Some(..)) => {
```

Comment:
This is just querying the tactic we end up using. We request `LimitedHorizontalVertical` above (although this is for function calls, not tuples).

For tuples, we use `format_item_list` which uses `HorizontalVertical`. I believe it is already tested in target/tuple.rs